.github/docker/rez-win-base/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# escape=`
+
+# Please Note: Any " requires \" in the Dockerfile for windows.
+
+# The windows version has to match the host system.
+# 1809 ->	10.0.17763.805 -> windows-2019
+# Compare: https://hub.docker.com/_/microsoft-windows-servercore
+ARG WINDOWS_VERSION=1809-amd64
+
+FROM mcr.microsoft.com/windows/servercore:$WINDOWS_VERSION
+
+SHELL ["powershell.exe", "-NoLogo", "-NoProfile", "-ExecutionPolicy", "ByPass"]
+
+# ------------------------------------------------------------------------------------------------------------
+# Chocolatey
+#
+ENV chocolateyUseWindowsCompression false
+RUN iex ((new-object net.webclient).DownloadString('https://chocolatey.org/install.ps1'))
+RUN choco feature disable --name showDownloadProgress
```

Comment:
We should do all the installs into one `RUN` command. That will save us some layers, and so reduce the image weight. Also, I suspect choco downloads files in a cache of some sort that will stay in the image if we don't remove this cache. We should make sure to clean any files that was created by choco. Note that the deletion of such files needs to be done in the same layer that they were created. So in the end, we will have one `RUN` that installs stuff and that also deletes stuff.

Reply:
I've merged runs where possible.
I've added a cleaning job at the end of the images. Not sure how much is saves vs. not having it - I believe choco does cleanup temporary downloads. It is a separate commit in case we want to revert it. The images are still somewhat big but I guess that is the the windows image that mostly counts.